src/resources/version.ts

```diff
@@ -153,6 +154,14 @@ class VersionResource extends CrudResource<typeof SVersion['schema'], ModelKey,
 
     return data;
   }
+
+  public async checkPrototypeSharedLogin(id: VersionID, body: { password: string }) {
+    this._assertModelID(id);
+
+    const { data } = await this.fetch.put<{ versionID: string }>(`${this._getCRUDEndpoint(id)}/prototype/share/login`, body);
+
+    return data;
```

Comment:
This triggers the endpoint to validate Prototype Share View authentication requests.